pkg/investigations/chgm/chgm.go

```diff
@@ -460,36 +411,12 @@ func extractUserDetails(cloudTrailEvent *string) (CloudTrailEventRaw, error) {
 	return res, nil
 }
 
-// PostLimitedSupport will put the cluster into limited support
-// As the deadmanssnitch operator is not silenced on limited support
-// the alert is updated with notes and silenced
-func postLimitedSupport(clusterID string, notes string, ocmCli ocm.Client, pdCli pagerduty.Client) error {
-	err := ocmCli.PostLimitedSupportReason(chgmLimitedSupport, clusterID)
-	if err != nil {
-		return fmt.Errorf("failed posting limited support reason: %w", err)
-	}
-	// we need to aditionally silence here, as dms service keeps firing
-	// even when we put in limited support
-	return pdCli.SilenceAlertWithNote(notes)
-}
-
-// HandleResolved removes limited support when a CHGM resolves
-func HandleResolved(r *investigation.Resources) error {
-	logging.Info("Remove 'Cluster has gone missing' limited support reason if any...")
-
-	removedReasons := false
-	err := utils.WithRetries(func() error {
-		var err error
-		removedReasons, err = r.OcmClient.DeleteLimitedSupportReasons(chgmLimitedSupport, r.Cluster.ID())
-		return err
-	})
+// postChgmSLAndSilence will send the CHGM SL and silence the alert
+func postChgmSLAndSilence(clusterID string, ocmCli ocm.Client, pdCli pagerduty.Client) error {
+	err := ocmCli.PostServiceLog(clusterID, &chgmSL)
 	if err != nil {
-		return fmt.Errorf("failed to remove limited support")
-	}
-
-	if removedReasons {
-		metrics.Inc(metrics.LimitedSupportLifted, r.AlertType.String(), r.PdClient.GetEventType(), chgmLimitedSupport.Summary)
+		return fmt.Errorf("failed sending service log: %w", err)
 	}
 
-	return nil
+	return pdCli.SilenceAlertWithNote("Customer stopped instances. Sending SL and silencing alert.")
```

Comment:
```suggestion
	return pdCli.SilenceAlertWithNote("Customer stopped instances. Sent SL and silencing alert.")
```

nit, since the SL has already been sent successfully here